examples/codesandbox/src/Banner.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { FormattedMessage } from "text-provider";
+
+export default class Banner extends React.Component {
+  /**
+   * The FormattedMessage component reads the string
+   * against the id provided from the TextProvider component
+   * defined in this components ancestory. The matching string
+   * is then rendered.
+   */
+  render() {
+    return (
+      <div className="App">
+        <h1>
+          <FormattedMessage id="title" />
+        </h1>
+        <h2>
+          <FormattedMessage id="subTitle" />
```

Comment:
Can you add an example where you use the `values` prop also ?

Reply:
Added.